draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -106,10 +107,10 @@
 
     <abstract>
       <t>
-        This document describes procedures that make use of Autonomous System Provider Authorization (ASPA) objects in the 
-		Resource Public Key Infrastructure (RPKI) to verify the Border Gateway Protocol (BGP) AS_PATH attribute of advertised routes.
-        This type of AS_PATH verification provides detection and mitigation of route leaks and improbable AS paths.
+        This document describes procedures that make use of Autonomous System Provider Authorization (ASPA) objects in the Resource Public Key Infrastructure (RPKI) to verify the Border Gateway Protocol (BGP) AS_PATH attribute of advertised routes.
+        This type of AS_PATH verification provides detection and mitigation of route leaks.
```

Comment:
I though we agreed that we are targeting not only route leaks.

Reply:
@mitradir 
I've restored the sentence to: 
"This type of AS_PATH verification provides detection and mitigation of route leaks and some forms of AS path manipulation."